storage/ballot_queue.go

```diff
@@ -845,6 +860,7 @@ func (s *Storage) MarkVerifiedResultsDone(processID []byte) error {
 	defer s.globalLock.Unlock()
 
 	// initialize the read transaction over the results prefix
+	defer util.HandleClosedDBPanic()
```

Comment:
umm do we really need all this stuff?

Reply:
Yes, it's not only an issue of this PR. When a sequencer stops and there are processes running that involve database operations executed when the database is closed, the sequencer panics.